src/unix/linux/x11/dl/libssl.h

```diff
@@ -157,6 +157,11 @@ static bool libssl_global_init(void)
 			library_init = true;
 		}
 
+		if (!LIBSSL_SO) {
+			LIBSSL_SO = MTY_SOLoad("libssl.so.3");
+			library_init = false;
```

Comment:
Is it what you intended to write?
```suggestion
			library_init = true;
```

Reply:
Nope, because library_init gets set to true due to trying libssl.so.1.0.0 first. Which needed the init, this version doesn't require the init.